module/module.go

```diff
@@ -175,6 +175,9 @@ type peerResourceState struct {
 
 // Module represents an external resource module that services components/services.
 type Module struct {
+	// The name of the module as per the robot config.
```

Comment:
```suggestion
	// The name of the module as indicated by `VIAM_MODULE_NAME`.
```
I realize it is the name from the config, but nice to note that it's derived from the env var?